pkg/operatorsource/datastore_test.go

```diff
@@ -27,5 +27,5 @@ func TestGetPackageIDs(t *testing.T) {
 	result := ds.GetPackageIDs()
 	actual := strings.Split(result, ",")
 
-	assert.EqualValues(t, expected, actual)
+	assert.Subset(t, expected, actual)
```

Comment:
This change needs to go into the commit where the bug was introduced given it has not been merged.